src/api/java/jni/cvc_Solver.h

```diff
@@ -0,0 +1,37 @@
+/* DO NOT EDIT THIS FILE - it is machine generated */
```

Comment:
It would be good to generate this file on the fly.

Reply:
Done